src/test/groovy/platform/edxPlatformMasterSpec.groovy

```diff
@@ -0,0 +1,56 @@
+package platform
+
+import groovy.util.slurpersupport.GPathResult
+import groovy.util.slurpersupport.Node
+import javaposse.jobdsl.plugin.JenkinsJobManagement
+import javaposse.jobdsl.dsl.GeneratedItems
+import javaposse.jobdsl.dsl.GeneratedJob
+import javaposse.jobdsl.dsl.DslScriptLoader
+import javaposse.jobdsl.dsl.JobManagement
+import org.junit.ClassRule
+import org.jvnet.hudson.test.JenkinsRule
+import spock.lang.Shared
+import spock.lang.Specification
+import spock.lang.Unroll
+
+
+class edxPlatformMasterJobSpec extends Specification {
+
+    @Shared
+    @ClassRule
+    JenkinsRule jenkinsRule = new JenkinsRule()
+
+    def @Shared DslScriptLoader loader
+    def @Shared JobManagement jm
+
+    /**
+    * Seed a DSL script and verify that the correct number of jobs are created, without throwing
+    * any exceptions
+    **/
+    @Unroll("test seeding #dslFile")
+    void 'test dsl creates correct xml'() {
+
+        setup:
+        JenkinsJobManagement jm = new JenkinsJobManagement(System.out, [:], new File('.'))
+        loader = new DslScriptLoader(jm)
+
+        when:
+        File dslScriptPath = new File("platform/jobs/${dslFile}")
+        GeneratedItems generatedItems = loader.runScript(dslScriptPath.text)
+
+        then:
+        noExceptionThrown()
+        generatedItems.jobs.size() == numJobs
+
+        where:
+        dslFile                                   | numJobs
+        'edxPlatformAccessibilityMaster.groovy'   | 5
+        'edxPlatformBokChoyMaster.groovy'         | 5
+        'edxPlatformJsMaster.groovy'              | 5
+        'edxPlatformLettuceMaster.groovy'         | 5
+        'edxPlatformPythonUnitTestsMaster.groovy' | 5
```

Comment:
What triggered the change in job counts?

Reply:
I had changed the test spec to focus on the Master jobs (which have different job counts) instead of the PR jobs. The reason for this was that the newer version of the GHPRB uses it's own syntax which is a lot harder to test using the Jenkins test harness